src/cloudWatchLogs/cloudWatchLogsUtils.ts

```diff
@@ -15,18 +17,40 @@ import { fromExtensionManifest } from '../shared/settings'
  * Destructures an awsCloudWatchLogs URI into its component pieces.
  * @param uri URI for a Cloudwatch Logs file
  */
-export function parseCloudWatchLogsUri(uri: vscode.Uri): { groupName: string; streamName: string; regionName: string } {
+export function parseCloudWatchLogsUri(uri: vscode.Uri): {
```

Comment:
That definitely cleans it up a bunch. Thank you! After making this change, I realized it made a lot more sense to move `streamName` into the `parameters` object rather than the `logGroupInfo`. This simplified this function even more.